tests/unit/utils/test_config_mlflow.py

```diff
@@ -37,3 +37,7 @@ def test_mlflow_console_config():
     clean_lineapy_env_var()
     for k, v in existing_lineapy_env.items():
         os.environ[k] = v
+
+    # Reset MLFlow options to not affect subsequent tests
+    options.set("mlflow_tracking_uri", None)
+    options.set("default_ml_models_storage_backend", None)
```

Comment:
Is this manual reset still necessary despite updates you made above in `tests/conftest.py`?

Reply:
If any test sets any `lineapy.options`, it should reset the flags otherwise any test running after the current test might query the `lineapy_config` inside `utils/config.py` which contains the cached configurations (attributes of `lineapy_config`), which cannot be unset by simply removing the env variables as done in `tests/conftest.py`. That's why `test_mlflow.py` also has the same resetting operation performed.

Reply:
Yeah, this makes sense, thanks for the clarification!